content/blog/intro-to-step-functions/index.md

```diff
@@ -136,7 +136,7 @@ nice_fn = lambda_.Function('niceFunction',
 
 Finally, we need to update the state machine with a new workflow that includes our new function. The state machine takes three arguments, a name, an IAM role, and the workflow as a JSON document. The code to get the ARN names is Pulumi specific and requires an explanation if you are not familiar with the [Pulumi programming model]({{< relref "/docs/intro/concepts/programming-model" >}}).
 
-Instance properties of a resource are called outputs, which behave much like [promises](https://en.wikipedia.org/wiki/Futures_and_promises). This method is necessary because outputs may not be fully available until a resource is fully provisioned. To get the value of a resource, Pulumi has the `apply` method, which has a call back that receives the raw value and computes a new output. `Apply` is how we get the lambda ARN names in the workflow document. The [Output.all]({{< relref "docs/intro/concepts/programming-model#outputs" >}}) method joins a list of Outputs that we can apply to a Python lambda.
+Instance properties of a resource are called outputs, which behave much like [promises](https://en.wikipedia.org/wiki/Futures_and_promises). This method is necessary because outputs may not be fully available until a resource is fully provisioned. To get the value of a resource, Pulumi has the `apply` method, which has a call back that receives the raw value and computes a new output. `Apply` is how we get the lambda ARN names in the workflow document. The [Output.all]({{< relref "/docs/intro/concepts/programming-model#outputs" >}}) method joins a list of Outputs that we can apply to a Python lambda.
```

Comment:
This is just adding `/` in front of the relref link to make it absolute to fix the Hugo build warning:

```
WARN 2020/02/19 13:03:33 make non-relative ref/relref page reference(s) in page %q absolute, e.g. {{< ref "/blog/my-post.md" >}}
```